client/src/components/payment/payment-summary.vue

```diff
@@ -50,107 +50,46 @@ import PaymentMixin from '@/components/payment/payment-mixin'
   }
 })
 export default class PaymentSummary extends Mixins(PaymentMixin) {
-  @Prop(Number) id: number
-  @Prop(Object) summary: any
-  @Prop(Object) receipt: any
-  @Prop({ default: {
-    receipt: {
-      created_on: '',
-      total: 0.00,
-      references: [{
-        receipt_number: '',
-        status_code: ''
-      }]
-    },
-    summary: {
-      completionDate: '',
-      statusCode: ''
-    }
-  } })
-  protected fetchError: string = ''
+  @Prop(Number)
```

Comment:
Curious about this. What is the significance of the type definition here if we are already doing that on the property itself?
Also, would these props be better off with a default?

Reply:
🤷

Reply:
TBH I don't know. I tried to not change this file beyond cleanup (since this was partly the file I used as a template for the new components).

Reply:
Fair enough. We can research but definitely no action required here!

Reply:
https://www.npmjs.com/package/vue-property-decorator#Prop